<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QMainWindow" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1070</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1070</width>
    <height>650</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dynamix note width changer GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainGUI.qrc">
    <normaloff>:/mainGUI/DNX icon.ico</normaloff>:/mainGUI/DNX icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadFile">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Chart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit queue:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveChart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="open_file_name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="widthMultiply">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Width multiplier</string>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>501</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="widthSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.01</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5.00</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="limitCheck">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Limited multiplier</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TimeSpecify">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Time</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start time(bar):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>End time(bar):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="stime">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="etime">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="startCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Set Start Time</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="endCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Set End Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>134</red>
          <green>29</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>134</red>
          <green>81</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>112</red>
          <green>110</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="sideChoose">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Sides</string>
    </property>
    <widget class="QCheckBox" name="checkMiddle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Middle</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkRight">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="browse">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="widthApply">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>APPLY CHANGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="loaded_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>841</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>112</red>
          <green>110</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>112</red>
          <green>110</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>18</red>
          <green>21</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>18</red>
          <green>21</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>107</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>109</red>
          <green>107</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>112</red>
          <green>110</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>112</red>
          <green>110</green>
          <blue>113</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dynamix Note Width Changer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="modeSelect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode Select</string>
    </property>
    <widget class="QRadioButton" name="active_multiplier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Width multiplier</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="active_randomizer1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Width randomizer 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="active_randomizer2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Width randomizer 2</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulanguages">
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>241</red>
          <green>239</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>214</red>
          <green>211</green>
          <blue>205</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>149</red>
          <green>147</green>
          <blue>151</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>224</red>
          <green>221</green>
          <blue>227</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>202</red>
          <green>201</green>
          <blue>193</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>233</red>
          <green>231</green>
          <blue>227</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>238</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="title">
     <string>languages</string>
    </property>
    <addaction name="actionEnglish_2"/>
    <addaction name="actionChinese"/>
   </widget>
   <addaction name="menulanguages"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish_2">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainGUI.qrc"/>
 </resources>
 <connections/>
</ui>
